python2verilog/ir/graph.py

```diff
@@ -42,8 +47,17 @@ class Element:
     """
 
     def __init__(self, unique_id: str, name: str = ""):
-        self._name = get_typed(name, str)
-        self._id = get_typed(unique_id, str)
+        self._name = get_typed_strict(name, str)
+        self._id = get_typed_strict(unique_id, str)
+
+    def nonclocked_children(self) -> Iterator[Element]:
+        """
+        Yields self and optimal nonclocked children of element
+
+        :return: [children_branch_0, children_branch_1, ...]
+        """
+        logging.debug(f"Non-over-written {type(self)}")
+        yield from ()
```

Comment:
What does this do? Does it just call a non-existent function and yield nothing?

Reply:
It's meant to be an abstract method that is overwritten in subclasses. However, a subclass not overwriting it is not a terrible error either.